blueocean-dashboard/src/main/js/creation/flow2/FlowStepStatus.js

```diff
@@ -5,8 +5,14 @@ const status = {
     ACTIVE: 'active',
     COMPLETE: 'complete',
     INCOMPLETE: 'incomplete',
+    ERROR: 'error',
 
-    values: () => [status.ACTIVE, status.COMPLETE, status.INCOMPLETE],
```

Comment:
Seems this functionality should go into a constructor if it's a pattern, or we want to make sure `values(`) stays correct over the life of the code? Something like:

```
const status = new FakeEnum({
    ACTIVE: 'active',
    COMPLETE: 'complete',
    INCOMPLETE: 'incomplete',
    ERROR: 'error',
});
```